src/main/java/alfred/action/Ui.java

```diff
@@ -0,0 +1,154 @@
+package alfred.action;
+
+import alfred.task.Task;
+import java.util.Scanner;
+/**
+ *
+ * Contains methods which prints messages Alfred could send
+ * to the users according to what the users input.
+ *
+ */
+public class Ui {
+    private static final String LOGO =
+            "    __      \n"
+            + "   /  \\    \n"
+            + "  / /\\ \\  \n"
+            + " / /__\\ \\  \n"
+            + "/_/    \\_\\ \n";
+
+    private static final String GREETING =
+            "Greetings, my name is Alfred. \n" + "How may I assist you?";
+
+    private static final String LIST = "Here is your list.";
+
+    private static final String BLAH = "Blah.";
+
+    private static final String BYE = "It has been great pleasure to be at your service. Goodbye.";
+
+    private static final String ADD = "Got it. I've added this task:";
+
+    private static final String DELETE = "Got it. I've removed this task:";
+
+    private static final String MARK = "You have successfully completed the following task:";
+
+    private static final String UNMARK = "You requested to mark the following task as undone:";
+
+    private static final String TASK_SIZE = "Here is the number of tasks in the list:";
+
+    private static final String WARNING = "Do be careful with your input or you have to re-run me again";
```

Comment:
I really like how the bot's reply are added into class variables, which makes it easier for the user to read to pre-determined replies of the bot instead of going into each method!